src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -0,0 +1,90 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+@Entity
+public class Recipe extends AbstractEntity {
+
+	private HashMap<String, String> ingredient;
+
+	private String name;
+	private int servings;
+	private int cookTime;
+	private int prepTime;
+	private String images;
```

Comment:
Because this is a single string maybe just image? Or, we could change it to a list if we want to include multiple images.